e2e/hive/docker-compose.yml

```diff
@@ -39,7 +42,20 @@ services:
       - ./warehouse:/warehouse
     logging:
       driver: none
+  hive:
+    image: metorikku/hive
+    environment:
+      - CONNECTION_URL=jdbc:mysql://hive-db:3306/hive?useSSL=false
+      - CONNECTION_USER_NAME=root
+      - CONNECTION_PASSWORD=pass
+      - WAREHOUSE_DIR=file:///warehouse
+      - WAIT_HOSTS=hive-db:3306
+    depends_on:
+      - hive-db
+    volumes:
+      - ./output/:/examples/output/
```

Comment:
Not needed, you need this:
- ./warehouse:/warehouse

Reply:
no need for volumes